samples/nextjs/scripts/scaffold-component.ts

```diff
@@ -98,6 +98,14 @@ if (manifestOutputPath) {
   console.log(`* Add the component to a route using Sitecore Experience Editor, and test it.`);
 }
 
+/**
+ * Force to use `crlf` line endings
+ * @param {string} content
+ */
+function editLineEndings(content: string) {
+  return content.replace(/\r|\n/gm, '\r\n');
```

Comment:
Is it obvious what's happening here or do we need to add something to the comment regarding why we're doing this?

Reply:
I extended the description a little bit, explained